### PR TITLE
WRQ-30274: Prepare for merging `feature/enact_v5` to `develop` branch

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We will merge `feature/enact_v5` branch to `develop` to release next major alpha version of agate.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Merged `feature/enact_v5` branch to `develop`
- Modified `npm-shrinkwrap.json` in sampler
- Changed ubuntu dist in travis.yml from `focal` to `jammy`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-30274

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Comment by vJIYEv** [comment]
unit test seems to be failed because of enact version.
It will be ok after merge enact branch `feature/enact_v5` to `develop` in https://github.com/enactjs/enact/pull/3261

File: samples/sampler/package.json [hunk 1]
```diff
@@ -24,18 +24,21 @@
     "@enact/core": "^4.9.0",
```
Comment (MikyungKim) [inline samples/sampler/package.json:24]: Should we bump up the version to 5.0.0-alpha.1?

File: samples/sampler/.qa-storybook/main.js [hunk 2]
```diff
@@ -1,23 +1,42 @@
...
 		disableTelemetry: true
 	},
 	features: {
 		postcss: false,
 		warnOnLegacyHierarchySeparator: false
 	},
-	framework: '@storybook/react',
+	framework: {
+		name: '@storybook/react-webpack5',
+		options: {}
```
Comment (MikyungKim) [inline samples/sampler/.qa-storybook/main.js:17]: Could you remove this empty options? I thought it was removed.

**Comment by MikyungKim** [comment]
@vJIYEv Could you bump up the enact dependencies to the latest in package.json files under this repo, please?

File: samples/sampler/.qa-storybook/main.js (hunk 2 above)
Comment (vJIYEv) [reply samples/sampler/.qa-storybook/main.js]: ok

File: samples/sampler/package.json (hunk 1 above)
Comment (vJIYEv) [reply samples/sampler/package.json]: ok

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]

**Pull request merged**